src/scripts/settings/webSettings.js

```diff
@@ -0,0 +1,17 @@
+define(['appStorage', 'events'], function (appStorage, events) {
+    'use strict';
+
+    var data = {};
+
+    function WebSettings() {
+        fetch("/config.json").then(function (response) {
+            data = response.json();
+        })
+    }
+
+    WebSettings.prototype.enableMultiServer = function () {
+        return data.multiServer || false;
```

Comment:
Since getting the config is async this could randomly return the default if the fetch has not completed when this function is called which doesn’t seem ideal. I wonder if it would be better to have a single function `getConfig()` or something that returns a `Promise` that will resolve with the complete config object.